.github/workflows/github-actions-test-lint.yml

```diff
@@ -16,5 +16,7 @@ jobs:
       run: pip install -e .[dev]
     - name: Running pylint
       run: pylint target_hdfs
+    - name: Running black formatter check
+      run: black --check --diff --skip-string-normalization .
```

Comment:
`--skip-string-normalization` param is to avoid conflict between the quotes convention in black and pylint.

> If you are adopting Black in a large project with pre-existing string conventions (like the popular [“single quotes for data, double quotes for human-readable strings”](https://stackoverflow.com/a/56190)), you can pass --skip-string-normalization on the command line. This is meant as an adoption helper, avoid using this for new projects.